### DISCUSSION TITLE
Promoting odudex to lead maintainer

### DISCUSSION THREAD

**Discussion by ghost** [selfcustody/krux#222] [Announcements]
Hey everyone,

For a while now I've been unable to devote the time to Krux that it needs. It's a promising project with a growing user base and deserves someone who can keep up with the rapid pace of development and user demands.

Odudex has been a long-time contributor to the project, tirelessly adding new features, promoting the project, and offering support to new users. On top of that, he understands the importance of open hardware and software for bitcoin. I know that he is more than well-suited to take over the reins as project lead and will do the necessary work to put out great new releases of Krux. 

23.07.0 will be quite the improvement when it comes out!

**Comment by tadeubas** [comment]
Totally agree!

**Comment by qlrd** [comment]
Agree!

**Comment by odudex** [comment]
Thank you for the words Jeff! I've been learning a lot and this project has given me many joys. I'll do my best to keep good standards, help the project grow consistently and welcome new Krux contributors like I was.
At any time, if you feel you can take the wheel again, we will all be very happy to have you back leading the project! If not, please come by eventually to give us some guidances.
